overlord/devicestate/export_test.go

```diff
@@ -481,7 +481,7 @@ func MockUserLookup(lookup func(username string) (*user.User, error)) (restore f
 var ProcessAutoImportAssertions = processAutoImportAssertions
 
 func MockCreateAllSystemUsers(createAllUsers func(state *state.State, assertDb asserts.RODatabase, model *asserts.Model, serial *asserts.Serial, sudoer bool) ([]*CreatedUser, error)) (restore func()) {
```

Comment:
nitpick: the name of this should match the name of the mocked thing: MockCreateAllKnownSystemUsers

Reply:
That is now renamed to correct name